run_instron_tests.bat

```diff
@@ -0,0 +1,19 @@
+@echo off
+REM Run all known tests using the IOC Testing Framework
```

Comment:
I've changed the comment, but I don't think we need to keep it. It's just convenient for now